docs/examples/Asymmetric Calculation Example.ipynb

```diff
@@ -0,0 +1,884 @@
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>778.966268</td>\n",
+       "      <td>1011.794128</td>\n",
+       "      <td>815.316511</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>1</th>\n",
+       "      <td>842.493451</td>\n",
+       "      <td>1121.320184</td>\n",
+       "      <td>815.316511</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2</th>\n",
+       "      <td>906.985877</td>\n",
+       "      <td>1239.243741</td>\n",
+       "      <td>815.316511</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>3</th>\n",
+       "      <td>972.433947</td>\n",
+       "      <td>1364.771826</td>\n",
+       "      <td>815.316511</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>4</th>\n",
+       "      <td>1038.839396</td>\n",
+       "      <td>1497.459012</td>\n",
+       "      <td>815.316511</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>5</th>\n",
+       "      <td>1106.212978</td>\n",
+       "      <td>1637.140796</td>\n",
+       "      <td>815.316511</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>6</th>\n",
+       "      <td>1174.572956</td>\n",
+       "      <td>1783.892394</td>\n",
+       "      <td>815.316511</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>7</th>\n",
+       "      <td>1243.944125</td>\n",
+       "      <td>1938.010800</td>\n",
+       "      <td>815.316511</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>8</th>\n",
+       "      <td>1314.357217</td>\n",
+       "      <td>2100.018933</td>\n",
+       "      <td>815.316511</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>9</th>\n",
+       "      <td>1385.848583</td>\n",
+       "      <td>2270.693567</td>\n",
+       "      <td>815.316511</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "             0            1           2\n",
+       "0   778.966268  1011.794128  815.316511\n",
+       "1   842.493451  1121.320184  815.316511\n",
+       "2   906.985877  1239.243741  815.316511\n",
+       "3   972.433947  1364.771826  815.316511\n",
+       "4  1038.839396  1497.459012  815.316511\n",
+       "5  1106.212978  1637.140796  815.316511\n",
+       "6  1174.572956  1783.892394  815.316511\n",
+       "7  1243.944125  1938.010800  815.316511\n",
+       "8  1314.357217  2100.018933  815.316511\n",
+       "9  1385.848583  2270.693567  815.316511"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "display(pd.DataFrame(output_data[\"line\"][\"i_from\"][:,0]))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "88950d8a-6cf2-4f4b-b6ec-0cd02fd19b29",
+   "metadata": {},
+   "source": [
+    "## Asymmetric State Estimation\n",
```

Comment:
i think you can re-use most of the components from before, right? that makes this a lot more readable. the symmetric one already contains a fully standalone example so this one only needs to show the diff